hw03_cruzmendeze.py

```diff
@@ -0,0 +1,141 @@
+#####################################################################################
+#
+# Author: Eric Cruz-Mendez
+# Username: cruzmendeze
+#
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Google Doc Link: https://docs.google.com/document/d/1P5qvQ1eb9AhFPyGV7J0JwgZ8XdeqU9qpbF82LSsu6yM/edit?usp=sharing
+#
+#####################################################################################
+
+import turtle
+turtle.colormode(255)  # Set colormode to RGB value
+
+# Function that makes the roof
+def roof(shape):
```

Comment:
The comment above the `def` belongs below the `def`, and formatted as a docstring (triple-quote string). 

Multiple instances of this throughout the code. Fix them all!

Reply:
I have corrected them to the specified format